docs/core/compatibility/core-libraries/6.0/y2038-incompatible.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Breaking change: .NET 6 is not supported on Year 2038 compatible Linux distros"
+description: Learn about the .NET 6 behavior on Year 2038 compatible Linxu distros.
+ms.date: 05/01/2024
+---
+# .NET 6 is not supported on Year 2038 compatible Arm32 Linux distros
```

Comment:
I'm a bit confused about this. It's not a breaking change introduced in .NET 6, so I don't really think it belongs here. @PriyaPurkayastha?

Reply:
Where would it otherwise go? Most of these breaks are of that character.

Here's a current customer example: https://github.com/dotnet/sdk/issues/40505

Perhaps the break is that we're not fixing this.